content/en/tracing/trace_explorer/visualize.md

```diff
@@ -0,0 +1,92 @@
+---
+title: Span Visualizations
+kind: documentation
+description: 'View spans in a list, or aggregate spans into timeseries, top lists and more.'
+further_reading:
+    - link: 'tracing/trace_explorer/'
+      tag: 'Documentation'
+      text: 'Trace Explorer'
+---
+
+## Overview
+
+Visualizations define how the queried span data is displayed. Select relevant visualizations to surface valuable information, such as a **list** for individual events, or as **timeseries** or **top lists** for aggregates. 
+
+## List view
+
+The list view displays a list of spans that match the selected context, defined by the [search bar query][1] filter and a [time range][2].
+
+In the **Show** row, choose which information of interest to display as columns. Manage the columns by either:
+
+- interacting with the table header row to **sort**, **rearrange**, or **remove** columns.
+- selecting a facet from the facet panel on the left, or from the trace side panel after clicking on a specific span, to **add** a column for a field. You can also add columns from with the **Options** button.
+
+{{< img src="tracing/trace_explorer/visualize/list_view_table_controls.mp4" alt="Configuring the display table" video=true style="width:80%;">}}
+
+{{< site-region region="us,eu,us3,us5,gov" >}}
```

Comment:
Is there a site that should be removed from this list? Or can we remove the site-region section tags?